src/com/content-games/games.js

```diff
@@ -152,14 +156,28 @@ export default class ContentGames extends Component {
 				return <ContentItemBox node={r.node} user={props.user} path={props.path} noevent={props.noevent ? props.noevent : null} />;
 			});
 
+			while (Games.length % layout !== 0) {
+				Games.push(<ContentItemBox placeHolder={true} />);
+			}
+
 			if ( !props.nomore /*|| added >= 10*/ ){
 				LoadMore = <ContentMore onclick={this.fetchMore} />;
 			}
 
+			const gridClass = `-columns-${layout}`;
+
 			return(
```

Comment:
Prefer
```
return (
```
(with a space between return and `(`).

I realize this is not a change from this set, just something to watch for.

Reply:
oops ofc